src/device/usbd.c

```diff
@@ -689,6 +784,8 @@ static bool process_get_descriptor(uint8_t rhport, tusb_control_request_t const
     case TUSB_DESC_CONFIGURATION:
     {
       tusb_desc_configuration_t const* desc_config = (tusb_desc_configuration_t const*) tud_descriptor_configuration_cb(desc_index);
+      TU_ASSERT(desc_config);

```

Comment:
TU_VERIFY ? This would be triggered by bad host behavior.

Reply:
`tud_descriptor_configuration_cb()` is callback implemented by application to provide configuration descriptor. We should assert if application return a NULL buffer.

Reply:
But, an application should return NULL when the host requests the descriptor for an invalid interface number?